src/datascience-ui/interactive-common/variableExplorer.tsx

```diff
@@ -182,31 +176,19 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
         if (!fastDeepEqual(this.props.variables, nextProps.variables)) {
             return true;
         }
-        if (
-            prevState.containerHeight !== this.state.containerHeight ||
-            (prevState.gridHeight !== this.state.gridHeight && !this.props.standaloneMode)
-        ) {
+        if (prevState.containerHeight !== this.state.containerHeight) {
```

Comment:
The real fix was above, the other changes here (maybe should have done this in a separate PR, but I was cleaning it up while working so I figured why not) are removing the few special cases used for when we hosted this control in a webview document versus the new single webview view. We now only have the webview view case.